tfhe/src/integer/gpu/server_key/radix/scalar_shift.rs

```diff
@@ -565,4 +565,76 @@ impl CudaServerKey {
         };
         stream.synchronize();
     }
+
+    /// # Safety
+    ///
+    /// - `stream` __must__ be synchronized to guarantee computation has finished, and inputs must
+    ///   not be dropped until stream is synchronised
+    pub unsafe fn unchecked_scalar_right_shift_logical_assign_async<Scalar, T>(
```

Comment:
Why this method is being added in this PR?

Reply:
It was convenient to add it, it's called in `unchecked_scalar_right_shift_logical_assign_async`.